scripts/ci/run-istio-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -v
+
+DIR="$(dirname "$0")"
+
+istio_setup() {
+        . "$DIR/install-minikube.sh" install
+        . "$DIR/install-minikube.sh" stop
+        . "$DIR/install-minikube.sh" start
+
+        . "$DIR/install-istio.sh" install
+        . "$DIR/install-istio.sh" stop
+        . "$DIR/install-istio.sh" start
+}
+
+helm_teardown() {
+	[ "$KEEP_RESOURCES" = "true" ] || . "$DIR/install-istio.sh" stop
+	[ "$KEEP_RESOURCES" = "true" ] || . "$DIR/install-minikube.sh" stop
+}
+
+. "$DIR/run-tests-utils.sh"
+network_setup
+istio_setup
+WITH_ISTIO=true
+TEST_PATTERN=Istio
```

Comment:
why not running k8s and istio in one single : run-k8s-tests

Reply:
please use only run-k8s-tests.sh

Reply:
Istio can also be used on systems w/o k8s,  (integrating with consul), for this reason it make sense to separate the tests.